### ISSUE TITLE
Automate creation of time series and oracle output target data

### ISSUE THREAD

**Issue by bsweger** [reichlab/variant-nowcast-hub#216]
**Background**

The variant-nowcast-hub script(s) to create the time series and oracle output formats of target data (#212 and #215) should be automated and scheduled to run after submissions are closed for each modeling round.

**Definition of done**

- [x] Both target data formats are created each week after round submissions are closed (fwiw, the repo has an [existing GitHub workflow that operates on this schedule](https://github.com/reichlab/variant-nowcast-hub/blob/main/.github/workflows/create-location-date-sequence-counts.yaml#L4))
- [x] Automation process should commit the new target data files and open a PR

Out of scope for this issue is automating the PR merge so we don't require a human approval (there's an [existing issue for that](https://github.com/reichlab/variant-nowcast-hub/issues/95))

Dependent on #212 and #215

**Comment by elray1** [comment]
There's a question here about the file format.  If the files are small (which we expect that they will be), we could use .csv rather than .parquet for better readability.

**Comment by bsweger** [comment]
> There's a question here about the file format. If the files are small (which we expect that they will be), we could use .csv rather than .parquet for better readability.

I moved the "write file" functionality to the #212 and #215 (because it made more sense to handle the writing as part of those issues). But to close the loop on the file format question, I decided to write .parquet files:

- To align with the format required for modelers
- To ensure that downstream processes can take full advantage of the partitioning we decided to use (unlike model-output files, target data files aren't transformed to parquet as they're synced to S3)

**Comment by bsweger** [comment]
Despite being in draft status, the attached PR is ready for review. Let's wait until after the holidays to deploy the updated workflow, since the changes are substantial, and I'd rather not run them for the first time on Christmas/New Year.
